### PR TITLE
Exclude T1 Event Grid live tests

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/eventgrid/ci.yml [hunk 1]
```diff
@@ -24,11 +22,9 @@ pr:
...
   paths:
     include:
-    - sdk/eventgrid/ci.yml
-    - sdk/eventgrid/Azure.Messaging.EventGrid/
-    - sdk/eventgrid/Microsoft.Azure.EventGrid/
-    - sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/
-    - sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/
+      - sdk/eventgrid/
+    exclude:
+      - sdk/eventgrid/Azure.ResourceManager.EventGrid/
```
Comment (jsquire) [inline sdk/eventgrid/ci.yml:27]: Wouldn't we also need to explicitly exclude the T1 paths as well?

File: sdk/eventgrid/service.projects [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup Condition="'$(SDKType)' == 'client'">
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.Messaging.EventGrid\**\*.csproj" />
-    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.EventGrid\**\*.csproj" />
+    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.EventGrid\**\*.csproj" Condition="'$(AZURE_EVENTGRID_TEST_MODE)' != 'Live'" />
```
Comment (jsquire) [inline sdk/eventgrid/service.projects:22]: Can we just split the pipeline and put the T1 package in its own?  Excluding only when it's a live run just seems to muddy things.

File: sdk/eventgrid/ci.yml [hunk 3]
```diff
@@ -24,11 +22,9 @@ pr:
...
 
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
```
[the remark this answers is not in the document]
Comment (JoshLove-msft) [reply sdk/eventgrid/ci.yml:30]: No, T1 uses the same ci file.

File: sdk/eventgrid/service.projects (hunk 2 above)
Comment (JoshLove-msft) [reply sdk/eventgrid/service.projects:22]: The goal here was to revert to the state prior to this PR - https://github.com/Azure/azure-sdk-for-net/pull/31927

I can look into just excluding the T1 path directly in the tests.yml.

Comment (JoshLove-msft) [reply sdk/eventgrid/service.projects:22]: Actually, I don't think there is a way to do it purely in tests.yml - the exclude condition controls what will trigger a pipeline run, not what a pipeline run will actually execute.

Comment (JoshLove-msft) [reply sdk/eventgrid/service.projects:22]: @weshaggard if you have any suggestions on how to implement this in a better way. I'm trying to get the EG test pipeline passing again and reverting to the previous state of this file.

Comment (jsquire) [reply sdk/eventgrid/service.projects:22]: I'd prefer that we split T1 and T2 into separate pipelines entirely.

**Review by jsquire — APPROVED** [review]

**Comment by JoshLove-msft** [comment]
@hallipr would it be possibly to get the Event Grid test pipeline (tests.yml) added to the live test dashboard?

**Pull request merged**